spec/client_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "spec_helper"
+require "teamsnap"
+
+RSpec.describe TeamSnap::Client do
```

Comment:
I actually think just having the block version would be best from a dev and client perspective. Just "one way" to do it.